criteria/geode/src/org/immutables/criteria/geode/GeodeQueryVisitor.java

```diff
@@ -148,14 +198,19 @@ private Oql binaryOperator(Call call) {
     } else {
       variable = constant.value();
     }
+    return variable;
+  }
 
+  private String getBindVariable(Object variable) {
```

Comment:
If this method mutates internal state (see `variables.add()`) I would call it differently (not just getter).

Reply:
good point, will rename